### DISCUSSION TITLE
Team communication

### DISCUSSION THREAD

**Discussion by inquinity** [Open-Lens/app#18] [General]
How can we communicate as a team? I haven't found a way to send messages with gh, other than discussions or issues. I want to contribute to the project and help get this tool back from the dead (or Mirantis), but I'm not really sure how to get started. Would a slack channel (or similar system) be worthwhile? Am I missing something else obvious (or unobvious) in gh?

**Comment by colintufts** [comment]
https://join.slack.com/t/openlensgroupe/shared_invite/zt-2ey77htas-p5TFVV_Q5GEW~k7DkyM4QA

**Reply by asinitson** [reply]
I have the same question as the topic author and the link below seems to be inactive. Did you guys end up joining Slack?

I am considering contributing too, but not sure where to start, so I would look around at first or maybe you have some suggestions. My background: Cloud/Infra/DevOps Engineer at the moment and software developer (mostly frontend - Angular, Typescript, Electron) in "past life".

**Reply by jkroepke** [reply]
the link has been expired.
